merchantapi/src/main/java/fi/hel/verkkokauppa/merchant/model/ConfigurationModel.java

```diff
@@ -0,0 +1,21 @@
+package fi.hel.verkkokauppa.merchant.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Data;
+import lombok.ToString;
+import org.springframework.data.elasticsearch.annotations.Field;
+import org.springframework.data.elasticsearch.annotations.FieldType;
+
+@Data
+@ToString
```

Comment:
@DaTa annotation should contain the @tostring generation also (https://projectlombok.org/features/Data) - no need to specify separate @tostring.

Reply:
Oh good catch, I copy pasted this and just added that @Data :D